.doc_gen/metadata/bedrock_metadata.yaml

```diff
@@ -0,0 +1,18 @@
+# zexi 0.4.0
+bedrock-ListFoundationModels:
+  title: List the available foundation models using an &AWS; SDK
```

Comment:
`List the available &BRlong; foundation models using an &AWS; SDK`

Reply:
Done